prisma/seeds/index.ts

```
@@ -12,7 +12,13 @@ import { seedAvatars } from './avatars.seed';
 import { seedStoryBuddies } from './story-buddies.seed';
 import { seedStories } from './stories.seed';
 
-const prisma = new PrismaClient();
+if (!process.env.DIRECT_DATABASE_URL) {
+  throw new Error('DIRECT_DATABASE_URL environment variable is required for seeding');
```

ESLint

[error] Replace `'DIRECT_DATABASE_URL·environment·variable·is·required·for·seeding'` with `⏎····'DIRECT_DATABASE_URL·environment·variable·is·required·for·seeding',⏎··`

(prettier/prettier)